### ISSUE TITLE
`@qualweb/core` Failed to resolve entry for package "@qualweb/core"

### ISSUE THREAD

**Issue by ruiaraujo012** [qualweb/qualweb#252]
Hello, I'm getting an error when using this packages in server components, this is the error that I get:
```
[vite] (ssr) Error when evaluating SSR module /Users/ruiaraujo/dev/gns/ama/a11y-test/app/routes/core.tsx?tsr-directive-use-server=: Failed to resolve entry for package "@qualweb/core". The package may have incorrect main/module/exports specified in its package.json: No known conditions for "." specifier in "@qualweb/core" package
```

**Comment by ruiaraujo012** [comment]
Locally, I've changed the `package.json` file in my `node_modules` to 
```
  "exports": {
    ".": {
      "import": {
        "types": "./dist/index.d.ts",
        "default": "./dist/index.js"
      },
      "require": {
        "types": "./dist/index.d.ts",
        "default": "./dist/index.js"
      }
    },
```

And now it's working locally, so I think it should be an easy fix?

**Comment by carlosapaduarte** [comment]
Fixed by #332
